sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
sensor_msgs/Image segmentation_image # binary segmentation
sensor_msgs/CameraInfo camera_info # camera info and intrinsics
sensor_msgs/PointCloud2 pointcloud # pointcloud of the scene in world frame
geometry_msgs/PoseStamped view_point # camera intrinsics
int16 number_of_candidates

---

BenchmarkGrasp[] grasp_candidates